### PR TITLE
Set up ice metrics

### PR DESCRIPTION
**Pull request by zhuker**
*Issue #, if available:*

merged master into `set-up-ice-metrics` @disa6302 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by zhuker** [comment]
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/issues/576

File: src/source/Metrics/Metrics.c [hunk 1]
```diff
@@ -4,56 +4,129 @@
...
 #include "../Include_i.h"
 
-
-STATUS getIceCandidatePairStats(PRtcPeerConnection pRtcPeerConnection, PRtcIceCandidatePairStats pRtcIceCandidatePairStats)
+STATUS getIceCandidatePairStats(PRtcPeerConnection pRtcPeerConnection,
+                                PRtcIceCandidatePairStats pRtcIceCandidatePairStats)
 {
     STATUS retStatus = STATUS_SUCCESS;
     PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
     CHK((pRtcPeerConnection != NULL || pRtcIceCandidatePairStats != NULL), STATUS_NULL_ARG);
```
Comment (MushMal) [inline src/source/Metrics/Metrics.c:12]: Minor: There is no need for the inner parenthesis

**Comment by zhuker** [comment]
This brings set-up-ice-metrics branch up to date with master

**Comment by disa6302** [comment]
@zhuker , 

I will merge this into awslabs:set-up-ice-metrics, but, I am splitting the different ICE metrics work into different branches to avoid a huge PR with lot of changes. Just FYI.

**Comment by zhuker** [comment]
> @zhuker ,
> 
> I will merge this into awslabs:set-up-ice-metrics, but, I am splitting the different ICE metrics work into different branches to avoid a huge PR with lot of changes. Just FYI.

do you mind sharing them as separate PRs?

**Comment by disa6302** [comment]
> > @zhuker ,
> > I will merge this into awslabs:set-up-ice-metrics, but, I am splitting the different ICE metrics work into different branches to avoid a huge PR with lot of changes. Just FYI.
> 
> do you mind sharing them as separate PRs?
> 
> @disa6302 how do we avoid duplicate work?

@zhuker , 
I will be setting up separate PRs. To avoid duplicate work, what we can do is put down our name against the metrics we are working on in #578. Let me update the ticket.

**Pull request merged**